force-app/main/default/aura/Employees_Comp_Table/Employees_Comp_Table.cmp

```diff
@@ -0,0 +1,93 @@
+                    <table class="slds-table slds-table_cell-buffer slds-table_bordered slds-table_striped">
+                        <thead>
+                            <tr class="slds-text-title_caps">
+                                <th class="slds-text-title_caps slds-text-align_center" scope="col">
+                                    <div class="slds-truncate" title="Name">Employee Name</div>
+                                </th>
+                                <th class="slds-text-title_caps slds-text-align_center" scope="col">
+                                    <div class="slds-truncate" title="Name">Birthdate</div>           
+                                </th>
+                                <th class="slds-text-title_caps slds-text-align_center" scope="col">
+                                    <div class="slds-truncate" title="Name">Job Category</div>
+                                </th>
+                                <th class="slds-text-title_caps slds-text-align_center" scope="col">
+                                    <div class="slds-truncate" title="Name">Compensation Type</div>
+                                </th>
+                                <th class="slds-text-title_caps slds-text-align_center" scope="col">
+                                    <div class="slds-truncate" title="Name">Amount</div>
+                                </th>
+                                <th class="slds-text-title_caps slds-text-align_center" scope="col">
+                                    <div class="slds-truncate" title="Name">Location</div>
+                                </th>
+                                <th class="slds-text-title_caps slds-text-align_center" scope="col">
+                                    <div class="slds-truncate" title="Name">Office</div>
+                                </th>
+                                <th class="slds-text-title_caps slds-text-align_center" scope="col">
+                                    <div class="slds-truncate" title="Name">Submitted</div>
+                                </th>
+                                <th class="slds-text-title_caps slds-text-align_center" scope="col">
+                                    <div class="slds-truncate" title="Name">Status</div>
+                                </th>
+                            </tr>
+                        </thead>
+                     <tbody>
+                        		<aura:iteration items="{! v.allCompensations}" var="compensation" aura:id="data">
+                                    <tr class="slds-text-align_center">
+                                        <th data-label="Employee Name" scope="row" class="slds-text-align_center" >
+                                            <lightning:button variant="base" label="{! compensation.Contact__r.Name}" title="Base action" onclick="{!c.editEmp}" value="{! compensation.Contact__r.Id}"/>
```

Comment:
You can move all of this code to another components, this will improve the re-usability of your code. Then, you just insert you component like this, ie: <c:yourComponentName compensation="{!item}"> </c:yourComponentName >

and into the new component you can declare the compensation as an attribute.

Reply:
Ok, i will change this.